src/qibocal/protocols/characterization/classification.py

```diff
@@ -351,17 +351,17 @@ def _plot(
     fig = make_subplots(
         rows=1,
         cols=len(models_name),
-        horizontal_spacing=SPACING * 3 / len(models_name),
+        horizontal_spacing=SPACING / len(models_name) * 3,
```

Comment:
With this, the benchmarks and the results plots are not aligned.
```suggestion
        horizontal_spacing=SPACING * 3/ len(models_name) * 3,
```

Reply:
Thank you